cookbook/upgrade/index.rst

```diff
@@ -0,0 +1,18 @@
+.. index::
+    single: Upgrading
+
+Upgrading
+=========
+
+So a new Symfony release has come out and you want to upgrade, great! Fortunately,
+because Symfony protects backwards-compatibility very closely, this *should*
+be quite easy.
+
+There are three types of upgrades, all needing a little different preparation:
+
+.. toctree::
+    :maxdepth: 2
+
+    /cookbook/upgrade/patch_version
+    /cookbook/upgrade/minor_version
+    /cookbook/upgrade/major_version
```

Comment:
Never mind, I guess the text above is only here to have some contents on the page that was used before, right?